server/www/components/vueResponsiveText/vue-responsive-text.js

```diff
@@ -0,0 +1,96 @@
+function getNodeSize(node, whatWeMeasure)// node : the html element we want to measure, whatWeMeasure : a String (either "width" or "height")
```

Comment:
This function could return an object `{width:0, height:0}` instead of calling it 2 times by changing `whatWeMeasure`.